core/src/main/scala/com/pingcap/tispark/TiDBOptions.scala

```diff
@@ -106,6 +111,8 @@ object TiDBOptions {
   val TIDB_DATABASE: String = newOption("database")
   val TIDB_TABLE: String = newOption("table")
   val TIDB_REPLACE: String = newOption("replace")
-  val TIDB_SKIP_COMMIT_SECONDARY_KEY = newOption("skipCommitSecondaryKey")
-  val TIDB_SAMPLE_FRACTION = newOption("sampleFraction")
+  val TIDB_SKIP_COMMIT_SECONDARY_KEY: String = newOption("skipCommitSecondaryKey")
+  val TIDB_REGION_SPLIT_NUM: String = newOption("regionSplitNum")
+  val TIDB_ENABLE_REGION_SPLIT: String = newOption("enableRegionPreSplit")
```

Comment:
add `pre` or not?

Reply:
Ther is no need to add `pre`. In TiDB,  `pre` means split region when creating table. We are better keep the same with TiDB.

Reply:
then change `enableRegionPreSplit` to `enableRegionSplit`

Reply:
done.